pkg/kubelet/kubelet.go

```diff
@@ -488,6 +489,9 @@ type Kubelet struct {
 
 	// Number of Pods which can be run by this Kubelet
 	pods int
+
+	// Monitor Kubelet's sync loop
+	syncLoopMonitor util.AtomicValue
```

Comment:
sync/atomic/Value does not exist in 1.3.
Till we support 1.3, we cant use that go type.